addons/l10n_es/data/res_country_group.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+	<record id="mainland_spain" model="res.country.group">
```

Comment:
We do `mainland_nl` for the Netherlands, let's do `mainlaind_es` here to be consistent (?)